tsuru/client/metadata.go

```diff
@@ -0,0 +1,263 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"sort"
+	"strings"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/go-tsuruclient/pkg/client"
+	"github.com/tsuru/go-tsuruclient/pkg/tsuru"
+	"github.com/tsuru/tsuru/cmd"
+	appTypes "github.com/tsuru/tsuru/types/app"
+)
+
+const metadataSetValidationMessage = `You must specify metadata in the form "NAME=value" with the specified type.
+
+Example:
+
+  tsuru app-metadata-set -t label NAME=value OTHER_NAME="value with spaces" ANOTHER_NAME='using single quotes'
```

Comment:
@pedrokiefer could you add `-a APPNAME` ?

Reply:
👍